llm-complete-guide/README.md

```diff
@@ -55,7 +55,40 @@ zenml secret create llm-complete --openai_api_key=<your-openai-api-key>
 export ZENML_PROJECT_SECRET_NAME=llm-complete
 ```
 
-### Setting up Supabase
+### Setting up Pinecone
+
+[Pinecone](https://www.pinecone.io/) is the default vector store used in this project. It's a cloud-native vector database that's optimized for machine learning applications. You'll need to create a Pinecone account and get an API key to use it.
+
+Once you have your Pinecone account set up, you'll need to store your API key and index name as a ZenML secret (with name `pinecone-zenml`). You can do this by running the following command:
+
+```shell
+zenml secret create pinecone-zenml --pinecone_api_key=<YOUR_PINECONE_API_KEY> --pinecone_env=<YOUR_PINECONE_ENV> --pinecone_index=<YOUR_INDEX_NAME>
```

Comment:
What should I use for pinecone_env

Reply:
I kept this to make the staging prod thing easier but it's not required right now and could be anything